repository/src/main/java/org/alfresco/repo/jscript/People.java

```diff
@@ -581,7 +581,7 @@ public Scriptable getPeoplePaging(String filter, ScriptPagingDetails pagingReque
     protected List<PersonInfo> getPeopleImpl(String filter, ScriptPagingDetails pagingRequest, String sortBy, Boolean sortAsc)
     {
         ParameterCheck.mandatory("pagingRequest", pagingRequest);
-        
+

```

Comment:
It would be best to completely reset this file to the original (master) state so it doesn't pollute the PR.